packages/components/tests/dummy/app/styles/pages/db-button-set.scss

```diff
@@ -0,0 +1 @@
+// BUTTON-SET
```

Comment:
if we don't need it, we can remove it and remove the import from the `packages/components/tests/dummy/app/styles/app.scss` file